Memoria/Defensa/defensa_minimales.tex

```diff
@@ -0,0 +1,388 @@
+            \}.
+    \end{align*}
+    \pause
+    \begin{exampleblock}{Teorema}
+        El espacio $\mathcal{H}(\Q^d, \Q^s)$ es denso en el espacio $\rrnnmc$. 
+    \end{exampleblock}
+\end{frame}
+
+%% ¿Cómo podemos encontrar la red neuronal concreta 
+\begin{frame}
+\frametitle{Aprendizaje} 
+\begin{enumerate}
+    \item \textbf{¿Cuánto de cerca estamos de la función ideal?}: Error cuadrático medio. 
+    \begin{equation*}
+        E_{\mathcal{D}}(h) = \frac{1}{N} \sum_{(x,y) \in \mathcal{D}} \sum_{k=1}^s(h_k(x)- y_k)^2. 
+    \end{equation*}
+    \pause 
+   \item  \textbf{Minimizar el error}: Descenso de gradiente. 
+    \begin{equation*}
+        h_{t+1}  = h_t - \eta \nabla E_{\mathcal{D}}(h_t).
+    \end{equation*}  
+    \pause
+    ¿Cómo determinar  \textcolor{red}{$h_0$}? 
+    ¿\textit{backbones}?
+\end{enumerate}
+    
+
+\end{frame}
+\subsection{Método inicialización aprendida}
+\begin{frame}{Método inicialización aprendida}
+    Algoritmo: 
+    \begin{enumerate}
+        \item Se toma $p \in \R^d$  aleatorio. 
+        \item Se extrae un subconjunto aleatorio  $\Lambda \subset \mathcal{D}$
+        que satisface 
+        \begin{equation*}
+            p \cdot (x_i-x_j) \neq 0
+        \end{equation*}
+        para cualquier $x_i,x_j$ distintos que estén en algún par de $\Lambda$.
+        \item Se construye de manera recursiva la red neuronal resolviendo esta sistema de ecuaciones: 
+        \begin{equation*}
+            \left\{ 
+                \begin{array}{l}
+                    S_{k} =
+                        M -  \frac{2 M}{p \cdot (x_k - x_{k-1})}(p \cdot x_{k})
+                    \\
+                    A_{k i} = \frac{2 M}{p \cdot (x_k - x_{k-1})}
+                    p_{i} 
+                    \\
+                    B_{* k} = y_k - y_{k-1}
+                \end{array}
+            \right.
+        \end{equation*} 
+\end{enumerate}
+ 
+\end{frame}
+
+\subsection{Equivalencia redes neuronales}
+\begin{frame}{Práctico}
+    Nuestra propuesta: 
+    \begin{align*}
+                h_k(x) = 
+                %\textcolor{teal}{\underline{\hspace{0.4cm}}}
+                \sum_{i=1}^{n} \textcolor{teal}{\beta_{i k}} \gamma_{i}( A_{i}(x))
+    \end{align*}
+    Modelo usual:
+    \begin{align*}
+        y_k(x) &= 
+        \textcolor{red}{\theta_k}
+        \left( 
+            \sum^n_{j=1} \textcolor{red}{A_{k}}
+            \left(
+                \sigma 
+                \left(
+                    A^{d}_{j k}
+                    \left(
+                        x
+                    \right)
+                \right)
+            \right)
+        \right)
+    \end{align*}
+    Solventamos diferencias: 
+    \begin{itemize}
+        \item $\theta$ eliminar observando convergencia en espacios $L_p$.
+        \item ¿Se puede eliminar el sesgo?
+    \end{itemize}
+\end{frame}
+
+%%%%% Sobre las funciones de activación
+\section{Democratización de las funciones de activación} 
```

Comment:
Concreta un poco cómo las vas a democratizar.

Reply:
No sé si te refieres al título o al contenido.
Si es sobre el título la verdad que quedaría más explicativo cambiarlo.
Si es sobre el contenido tenía pensado decirlo de palabra, cuando os haga la defensa si pensáis que sería conveniente dejarlo por escrito lo añado.

Reply:
Título